ngsPETSc/utils/firedrake.py

```diff
@@ -172,7 +173,7 @@ def createFromTopology(self, topology, name):
         cell = topology.ufl_cell()
         geometric_dim = topology.topology_dm.getCoordinateDim()
         cell = cell.reconstruct(geometric_dimension=geometric_dim)
-        element = ufl.VectorElement("Lagrange", cell, 1)
+        element = finat.ufl.VectorElement("Lagrange", cell, 1)
```

Comment:
```suggestion
        element = firedrake.VectorElement("Lagrange", cell, 1)
```
I think that would be more API stable.